### DISCUSSION TITLE
Two types of pinned tabs -> current workspace and across all workspaces

### DISCUSSION THREAD

**Discussion by xanthos84** [zen-browser/desktop#2405] [Ideas]
Someone put this into a closed discussion, so I post it as a new feature request:
----
I think it would be good to have two different functions:

1. Pinned tabs: Tabs are pinned on the current workspace only
2. Starred tabs: Tabs appear across all workspaces.

Probably position/order them as: Starred, Pinned, then Other tabs.

**Comment by mark-rifkin** [comment]
Just bumping this feature request again -- having per-workspace pinned tabs that display as normal tabs is a fantastic Arc feature!

**Comment by claymorwan** [comment]
i pray for this to be implemented I need it

**Comment by NyllRE** [comment]
Now there is with "Essential Tabs".
Zen browser continues to prove that it's the best browser of all time
